frontend/src/components/Login.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+    let active = false;
+
+    function isActive() {
+        return active ? "is-active" : "";
+    }
+</script>
+
+<style>
+    .modal-content {
+        width: 31.25em;
+        height: 35.5em;
+        background: #D9D9D9;
+        border-radius: 7px;
+    }
+
+    .link:hover {
+        text-decoration: underline;
+        color: #276BB0;
+    }
+
+    #login-contents {
+        margin: 2.5em 3.125em 0em;
+    }
+</style>
+
+<button class="button is-light" on:click={() => { active = true; }}>
```

Comment:
I still think the is-light is way too bold, imo is-primary looks better (it's common practice for websites to emphasise Sign Up, and have log in adjacent but less engaging).

Reply:
sounds good to me